### ISSUE TITLE
Can u add a separate message box for adding a shortcut in server connector?

### ISSUE THREAD

**Issue by astrojade** [EternalARK/BombSquad_Aeon_Mods#1]
Hi, if this is the wrong github page, for posting this issue, then i am sorry in advance. 
The issue is regarding the server connector UI. Is it possible if you can add a message box for the "Adding a shortcut checkbox" ?
That way, everyone can add a shortcut in the new tab directly, without having to access the manual tab,then go back to favourites tab, and then go back to manual tab to hit the submit button.
Just like how the rename shortcut has its own message box, for renaming any ip in the favourites tab. In the same way, please add a separate message box for the "adding a shortcut" check box. 
Example:
![image](https://user-images.githubusercontent.com/67503088/89401083-64574900-d732-11ea-88c6-bc88a118cfed.png)

Oh yes, one more thing, these server connector mods are interfering with the tournament games, causing it to fail due to modification of the game. Please do something about it.
P.S. what is the use of the interactive mode checkbox?

**Comment by EternalARK** [comment]
Yes. It's the right place to send your questions and feedbacks. 
First, _interfering with the tournament games, causing it to fail due to modification of the game._ is not caused by any of these mods. Mine worked properly. In most cases it's due to you have bad history in tournaments. Second, I tend to write the most efficient code which means I don't want to create a whole new popup just for add server,(Rename and other input windows use same Class, but I will be forced to add a port input box to achieve your proposal.) causing larger pack size and slow down in programs. But I will consider to add a help option to solve the use manual. (Or I will submit to your proposal if too many people are requesting for that.)
Anyway I will add a translation module really soon. That will help I guess..

**Comment by astrojade** [comment]
"In most cases it's due to you have bad history in tournaments". It is not just me who is having the same problem, I can screen record and show it to you, why this is happening. I doubt its a fault from your side because any modification to the game itself, is counted as tournament modification, when before, server related modifications wouldnt mess around with the tournament files. 
Do i need to prove it? Its not that hard for me to screen record.

**Comment by astrojade** [comment]
Ok, i found out what is wrong, technically even after deleting your files, we still need to uncheck your plugins to be not counted as modifications. Can you just do something, like the moment the files get deleted from the mod folder, the plugins get disabled too?Why is there a need to disable the plugins, when the files don't exist in the first place. Im sending a video regarding this.

**Comment by EternalARK** [comment]
Use mod manager to delete. That will help you to disable mods.
It's Eric who added new tournaments policies against any plugins I guess.
